apps/emqx/src/emqx_broker_sup.erl

```diff
@@ -19,13 +19,29 @@
 -behaviour(supervisor).
 
 -export([start_link/0]).
+-export([get_broker_pool_workers/0]).
 
 -export([init/1]).
 
+-define(broker_pool_sup, broker_pool_sup).
+
 start_link() ->
     ok = mria:wait_for_tables(emqx_shared_sub:create_tables()),
     supervisor:start_link({local, ?MODULE}, ?MODULE, []).
 
+get_broker_pool_workers() ->
```

Comment:
get processes from grpoc_pool:workers
supervisor:which_children is a gen_call

Reply:
:checkered_flag: